ui/app/AppLayouts/Profile/popups/TokenListPopup.qml

```diff
@@ -59,10 +59,7 @@ StatusDialog {
             CustomHeaderDelegate {}
         }
         delegate: CustomDelegate {}
-        /* This late binding has been added here because without it all
```

Comment:
can we please still keep this component, we are stilling assigning model on Component.onCompleted and not normally to avoid performance issues.